packages/backup-restore-operator/charts/templates/s3-secret.yaml

```
@@ -0,0 +1,30 @@
+{{- if .Values.s3.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "backupRestore.s3SecretName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "backupRestore.labels" . | nindent 4 }}
+type: Opaque
+stringData:
+  {{- with .Values.s3}}
+  {{- if .credentialSecretName }}
+  credentialSecretName: {{ .credentialSecretName }} 
+  {{- end }}
+  {{- if .region }}
+  region: {{ .region }}
+  {{- end }}
+  bucketName: {{ .bucketName }}
+  {{- if .folder }}
+  folder: {{ .folder }} 
+  {{- end }}
+  endpoint: {{ .endpoint }}
+  {{- if .endpointCA }} 
+  endpointCA: {{ .endpointCA }} 
+  {{- end }}
+  {{- if .insecureTLSSkipVerify }}
+  insecureTLSSkipVerify: {{ .insecureTLSSkipVerify }} 
+  {{- end }}
```

Should this be set to a default value or does the fact that it doesn't exist imply that it's false?

i.e.
```suggestion
  insecureTLSSkipVerify: {{ default false .insecureTLSSkipVerify }} 
```

---

yup the fact that it doesn't exist implies that it's false